content/doc/tutorials/tutorial-for-installing-jenkins-on-AWS.adoc

```diff
@@ -268,13 +268,20 @@ To download and install Jenkins:
 [ec2-user ~]$ sudo yum upgrade
 ----
 
-. Install Java:
+. Install Java (Amazon Linux 2):
 +
 [source,bash]
 ----
 [ec2-user ~]$ sudo amazon-linux-extras install java-openjdk11 -y
 ----
 
+. Install Java (Amazon Linux 2023):
++
+[source,bash]
+----
+[ec2-user ~]$ sudo yum install java-11-amazon-corretto-headless -y
```

Comment:
Since the [default package manager for Amazon Linux 2023](https://docs.aws.amazon.com/linux/al2023/ug/package-management.html) is `dnf`, would you optionally consider changing this to `dnf` instead of `yum`?

The `yum` package manager works on Amazon Linux 2023, but since they recommend `dnf` I think it would be good for us to use it in the example.
```suggestion
[ec2-user ~]$ sudo dnf install java-11-amazon-corretto-headless -y
```